admin_guide/sysctls.adoc

```diff
@@ -135,31 +136,49 @@ ifdef::openshift-origin[]
 endif::[]
 
 [[setting-sysctls-for-a-pod]]
-== Setting Sysctls for a Pod
+== Setting sysctls for a pod
 
-Sysctls are set on pods using annotations. They apply to all containers in the
-same pod.
+Sysctls are set on pods using the pod's securityContext. The securityContext
```

Comment:
Other references to securityContext take markup, `securityContext` or `*securityContext*`

Reply:
I think it's a parameter, so `securityContext` is correct.